mollie/api/resources/customer_mandates.py

```diff
@@ -8,7 +8,7 @@ class CustomerMandates(Base):
     customer_id = None
 
     def get_resource_object(self, result):
-        return Mandate(result, self)
+        return Mandate(result, client=self.client)
```

Comment:
Shouldn't this be `return Mandate(result, client)` since you've removed `client=` at all the other `get_resource_object()`?